.changeset/nine-penguins-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+Always invoke an injected Web3 Wallet provider, if any, to allow the user to continue the flow rather than be a no-op
```

Comment:
```suggestion
Bug fix: When the requested web3 provider cannot be found use any other available injected Web3 Wallet provider, instead of blocking the sign-in/sign-up flow.
```